lexers/rpmspec.lua

```diff
@@ -17,15 +17,23 @@ lex:add_rule('string', token(lexer.STRING, lexer.range('"')))
 
 -- Keywords.
 lex:add_rule('keyword', token(lexer.KEYWORD, word_match{
-  'Prereq', 'Summary', 'Name', 'Version', 'Packager', 'Requires', 'Recommends', 'Suggests',
-  'Supplements', 'Enhances', 'Icon', 'URL', 'Source', 'Patch', 'Prefix', 'Packager', 'Group',
-  'License', 'Release', 'BuildRoot', 'Distribution', 'Vendor', 'Provides', 'ExclusiveArch',
-  'ExcludeArch', 'ExclusiveOS', 'Obsoletes', 'BuildArch', 'BuildArchitectures', 'BuildRequires',
-  'BuildConflicts', 'BuildPreReq', 'Conflicts', 'AutoRequires', 'AutoReq', 'AutoReqProv',
-  'AutoProv', 'Epoch'
-}))
+    'Prereq', 'Summary', 'Name', 'Version', 'Packager',
+    'Requires', 'Recommends', 'Suggests', 'Supplements',
+    'Enhances', 'Icon', 'URL', 'Prefix', 'Packager', 'Group',
+    'License', 'Release', 'BuildRoot', 'Distribution', 'Vendor',
+    'Provides', 'ExclusiveArch', 'ExcludeArch', 'ExclusiveOS',
+    'Obsoletes', 'BuildArch', 'BuildArchitectures', 'BuildRequires',
+    'BuildConflicts', 'BuildPreReq', 'Conflicts', 'AutoRequires',
+    'AutoReq', 'AutoReqProv', 'AutoProv', 'Epoch'
+}) + (P('Patch') + P('Source')) * R('09')^0)
```

Comment:
You are excluding these additional keywords from the `token()` call. Move the `)` from `})` to the end of the line and it should work.

Reply:
Fixed in 9d0129edba37fba708969b46c779c8764e911348.